Rakefile

```diff
@@ -0,0 +1,53 @@
+#
```

Comment:
This looks like a duplicate of chef-powershell/Rakefile

Reply:
removed